shared/js/initApp.js

```diff
@@ -65,7 +65,9 @@ window.apps = {
       },
       onContinue: function() {
         if (lastServerResponse.videoInfo) {
-          showVideoDialog(lastServerResponse.videoInfo);
+          if (dashboard.clientState.hasSeenVideo(lastServerResponse.videoInfo.key)) {
```

Comment:
Is this the only caller of showVideoDialog? Looking, it appears that there are other call sites that may need to be updated.